rllib/utils/torch_utils.py

```diff
@@ -183,9 +186,11 @@ def mapping(item):
         # Everything else: Convert to numpy, then wrap as torch tensor.
         else:
             tensor = torch.from_numpy(np.asarray(item))
+
         # Floatify all float64 tensors.
-        if tensor.dtype == torch.double:
+        if tensor.is_floating_point():
```

Comment:
any chance this is a new api? if yes, maybe make sure it works with the version we pin for core.

Reply:
good point. I checked quickly and it's even there on torch=1.8.0 Where do we specify the torch pinned version in ray?
https://pytorch.org/docs/1.8.0/search.html?q=IS_FLOATING_POINT&check_keywords=yes&area=default